src/app/components/observation/observation/observation.component.html

```diff
@@ -18,7 +18,7 @@
 
   <!-- lazy-preload-prev-next="2" -->
   @for (attachment of attachments(); track attachment.AttachmentId) {
-    <swiper-slide (click)="imageClicked(attachment)" lazy="true">
+    <swiper-slide (click)="imageClicked(attachment.Url)">
```

Comment:
Hvis vi bruker $index i stedet for attachment.Url og sender det til modalen, så kan noe av logikken i observation-image-carousel.component.ts fjernes?